azure-mgmt-compute/src/test/java/com/microsoft/azure/management/compute/VirtualMachineOperationsTests.java

```diff
@@ -244,7 +244,7 @@ public void canCreateUpdatePriorityAndPrice() throws Exception {
             // not run to assert
             Assert.assertEquals(VirtualMachinePriorityTypes.REGULAR, foundVM.priority());
             Assert.fail();
-        } catch (CloudException e) {} // cannot change priority from spot to regular
+        } catch (CloudException e) {} // cannot change priority from low to regular
```

Comment:
Please confirm this behavior with compute service team

Reply:
This is written by service team, and my test also fits their statement.